### ISSUE TITLE
Level mapping for SBC CSV import schema

### ISSUE THREAD

**Issue by johnwdubois** [johnwdubois/rezonator#557]
**Background**
For importing the SBC CSV files into Rezonator, it's import to get the Levels right. The correct mappings are specified below.

**What do do**
Use the following information for mapping Levels in the import schema for SBC CSV (and for any similar files):
1. Since the SBC CSV file is organized as "one token per line" (OTPL), you can treat EVERY field in this file as a Token-level field. Thus, there is no need to list the Token fields separately - this is simply the default mapping.  ALL the fields in this type of CSV should get mapped to the Rezonator ImportToken grid (or whatever it's called).
2. In addition, for the SBC CSV there are exactly 7 CSV fields that should ALSO be mapped at the Unit level. Here they are: 
- discoId, unitId, agent (= Speaker or Participant), unitStart, unitEnd, unitLen, turnId.
3. Finally, there is exactly one field that can be mapped at 3 different levels, namely token, unit, and discourse. This field is: DiscoId. 
4. To handle this using the current implementation of the Rezonator import screen, we can simply implement a rule like the following:
    • If a field's Level is marked as Unit, it is mapped to BOTH Unit and Token levels.
    • If a field's Level is marked as Token, it is mapped ONLY to the Token level.
    • If a field's Level is marked as Discourse, it is mapped to the Discourse, Unit, and Token levels.
In other words, there's a hierarchy: Discourse implies Unit, and Unit implies Token (but not vice versa).

**Additional context**
1. Obviously, mapping the same field to more than one Level creates some redundancy in the grids. But redundancy is not a real problem, and this should make it easy to query/filter the data, and to match up all the user annotation with the user's original data.
2. Since we don't have a Discourse grid at the moment, we don't need to worry too much about the mapping of this Level. But in principle it should be part of the schema, to prepare for the existence of a Discourse grid in the future.

**Comment by Georgio-Klironomos** [comment]
@johnwdubois Currently how the import process works is that fields can only be assigned one level. So either this ticket implies a new major development for 0.9, or we're confused by what it is asking for. Hopefully the dev team can get an understanding of this ticket during a meeting tomorrow, Monday 8/31

**Comment by Georgio-Klironomos** [comment]
@johnwdubois SBC CSV's are now imported based on the rules specified in this ticket. It was also user tested today, and the issue found during the test has been fixed.
